docs/configuration.rst

```diff
@@ -84,4 +84,7 @@ Gunicorn configuration:
 
 Webapp configuration:
 
-To be documented.
+.. automoduleconfig:: crashstats.settings.base._config
```

Comment:
should this be under webapp?
```suggestion
.. automoduleconfig:: webapp.crashstats.settings.base._config
```

Reply:
Mmm... I want to say no it shouldn't because `webapp/` shouldn't be a Python module. But it turns out that it has a `__init__.py` file, so it is a Python module so maybe that works? The more correct way would be to do `crashstats.settings.base._config` and have `webapp/` be part of the Python path.

It should be easy to test. If you run `make docs` and it complains, then that's the wrong thing.

Reply:
make docs only succeeds when i use `webapp.`, but even then it's outputting an empty configuration table when i try it:
<img width="383" alt="Screenshot 2024-01-30 at 1 51 51 PM" src="https://github.com/mozilla-services/socorro/assets/433717/c612158b-89a3-4696-ba9b-a2cb1ab63503">

Reply:
this fixed it, tyty